pkg/ingester/client/compat.go

```diff
@@ -87,7 +87,7 @@ func MatrixFromSeriesSet(set storage.SeriesSet) model.Matrix {
 		m = append(m, &ss)
 	}
 
-	return m
+	return m, set.Err()
```

Comment:
Do we need to check `it.Err()` inside the for loop?

Reply:
Oh i see.. good catch!

Reply:
Do we? i think if err occurred the `it.Next()` would return false, and we would go to the return line

Reply:
Umm, will `set.Err()` return the same error as `it.Err()`?
I think `set.Err()` just returns nothing

Reply:
I am going to merge since it is used in test only

Reply:
No.. im fixing it

Reply:
done